releasenotes/notes/deprecate-complex-amp-41381bd9722bc878.yaml

```
@@ -0,0 +1,17 @@
+---
+deprecations:
+  - |
+    Initializing a :class:`~qiskit.pulse.library.ScalableSymbolicPulse` with complex `amp` value is now deprecated.
+    Particularly, the following pulses
+
+      * :class:`~qiskit.pulse.library.Gaussian`
+      * :class:`~qiskit.pulse.library.GaussianSquare`
+      * :class:`~qiskit.pulse.library.Drag`
+      * :class:`~qiskit.pulse.library.Constant`
+
+    initialize a :class:`~qiskit.pulse.library.ScalableSymbolicPulse`, and therefore initializing them with complex
+    `amp` is now deprecated.
+
+    Instead, one should use two floats for the `amp` and `angle` parameters, where `amp` represents the
+    magnitude of the complex amplitude, and `angle` represents the angle of the complex amplitude. i.e. the
+    complex amplitude is given by `amp * exp(1j * angle)`.
```

```suggestion
    complex amplitude is given by `amp` * exp(1j * `angle`).
```
I think italic math is hard to read.

---

Thanks.